man/org.freedesktop.hostname1.xml

```diff
@@ -287,6 +287,17 @@ node /org/freedesktop/hostname1 {
     purpose of those properties is to allow remote clients to access this information over D-Bus. Local
     clients can access the information directly.</para>
 
+    <para><varname>MachineID</varname> expose the 128bit machine ID, see
+    <citerefentry><refentrytitle>machine-id</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
+    details.</para>
+
+    <para><varname>BootID</varname> expose the 128bit boot ID, as per
+    <filename>/proc/sys/kernel/random/boot_id</filename>.</para>
+
+    <para><varname>VSockContextIdentifier</varname> exposes the system's local <constant>AF_VSOCK</constant>
+    CID fo the system, if it runs in a vsock-capable virtual machine environment. See <citerefentry project="man-pages"><refentrytitle>vsock</refentrytitle><manvolnum>7</manvolnum></citerefentry> for
```

Comment:
CID => connection ID

Reply:
"if it runs in a vsock-capable virtual machine environment" => "if one is available in the virtual machine environment"?